### ISSUE TITLE
Not moving with keybord

### ISSUE THREAD

**Issue by gsabanti** [daltoniam/DCCommentView#3]
DCCommentView is not moving up when it is tapped.

Code: 
        commentView = DCCommentView(scrollView: self.tableView, frame: self.view.bounds)
        self.view.addSubview(commentView)
        commentView.delegate = self
        commentView.charLimit = 255
        self.commentView.tintColor = GSColor.main

**Comment by daltoniam** [comment]
yeah I don't know why that is based off your code. Like I said in #2, I haven't looked or updated this code in years. That being said, I don't have any plans to look at this code in the near future, but a PR is welcomed if you find the cause. Best of luck!
